songpal/discovery.py

```diff
@@ -1,5 +1,5 @@
 import logging
-from async_upnp_client.discovery import async_discover
+from async_upnp_client.search import async_search
```

Comment:
Hmm, I thought this got already merged?

Reply:
Yeah, my git-fu is a little weak, but this is *hopefully* fixed now?